test/onnx/test_fx_op_consistency.py

```diff
@@ -1548,6 +1538,11 @@ def skip_torchlib_forward_compatibility(
             "Reshape", "empty tensor"
         ),
     ),
+    xfail(
+        "nn.functional.instance_norm",
+        model_type=pytorch_test_common.TorchModelType.TORCH_EXPORT_EXPORTEDPROGRAM,
+        reason="fixme: Assertion error: result mismatch",
```

Comment:
Is it because decomp slipping does not work with `ExportedProgram`, it fails?

Reply:
correct

Reply:
Well, to be more precise, decomp skipping were not enabled during `torch.export`. It is only enabled during `torch.onnx.dynamo_export`.